tests/testthat/helper-HTML.R

```diff
@@ -77,10 +77,14 @@ get_grid_lines <- function(html, p_name, grid_class){
   return(list(hor=attr_h, vert=attr_v))
 }
 # Function to send a key event
-sendKey <- function(key, code, keyCode) {
+sendKey <- function(key) {
   types <- c("keyDown", "keyUp")
+  key2code <- c(
+    Backspace=8,
```

Comment:
hi @siddhesh195 can you please add a comment here about where these key code numbers come from?

Reply:
commented for key-codes

Reply:
thanks, I see hex codes on https://learn.microsoft.com/en-us/windows/win32/inputdev/virtual-key-codes can you please add a comment about how you converted to decimal? is there a R command you used? or a different web page?

Reply:
I could not find the Microsoft web page with decimal values of codes. I modified the comment to say we use the corresponding decimal values of the key codes  given in the hex value in the web page. since it is a standard mathematical concept, do we need to really mention how we converted hex to decimal ? I can add a link of online converter but it might not work in the future.

Reply:
so ...
chromote dispatchKeyEvent requires these key codes in decimal, for the windowsVirtualKeyCode and nativeVirtualKeyCode arguments.
this is documented here https://chromedevtools.github.io/devtools-protocol/tot/Input/ 
```
windowsVirtualKeyCode
integer
Windows virtual key code (default: 0).
nativeVirtualKeyCode
integer
Native virtual key code (default: 0).
```
that page also says
```
code
string
Unique DOM defined string value for each physical key (e.g., 'KeyA') (default: "").
key
string
Unique DOM defined string value describing the meaning of the key in the context of active modifiers, keyboard layout, etc (e.g., 'AltGr') (default: "").
```
instead of that Microsoft page, probably this web page https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values is better to link, because it has the same info, plus the "KeyboardEvent.key Value" which is what we use for the key and code arguments,
* Backspace (0x08),
* Enter (0x0D)
* ArrowDown (0x28), 
The R code for conversion is:
```r
> strtoi(c("08","0D","28"),base=16)
[1]  8 13 40
```

Reply:
thank you for the link. I could not find that page using a simple google search.